pkgs/development/python-modules/prometheus-pandas/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ipython
+, matplotlib
+, numpy
+, pandas
+, requests
+}:
+
+buildPythonPackage rec {
+    pname = "prometheus-pandas";
+    version = "0.3.2";
+
+    src = fetchPypi {
+      inherit pname version;
+      hash = "sha256-zysdlYPjvhextAfW4vvGN6mSPVUbWLV+AvQ1hGWCAbw=";
+    };
+
+    propagatedBuildInputs = [
+      ipython
+      matplotlib
+      numpy
+      pandas
+      requests
+    ];
+    # Tests fail because: "NameError: name 'get_ipython' is not defined"
+    doCheck = false;
```

```suggestion
    doCheck = false;
    
    pythonImportsCheck = [
      "prometheus_pandas"
    ];
```
Could you please add pythonImportsCheck for a simple test?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck-using-pythonimportscheck